sorc/build_ufs_coupled.sh

```diff
@@ -13,15 +13,17 @@ if [ $target = hera ]; then target=hera.intel ; fi
 if [ $target = orion ]; then target=orion.intel ; fi
 if [ $target = stampede ]; then target=stampede.intel ; fi
 
-MOD_PATH=$cwd/ufs_coupled.fd/modulefiles/$target
+MOD_PATH=$cwd/ufs_coupled.fd/modulefiles
 
 module purge 
 module use $MOD_PATH 
-module load fv3 
+module load ufs_${target}
 cd ufs_coupled.fd/
 if [[ -d build ]]; then rm -Rf build; fi
 if [[ -d GOCART ]]; then
```

Comment:
based on the https://github.com/NOAA-EMC/WW3/wiki/Commit-Queue the if statement for if the the GOCART directory exists is not going to work by the time the update for the ice history is merged in.

Reply:
I'll change this back to using the -a switch. Unless after ufs-community/ufs-weather-model/pull/591 there will be a DAPP option that incorporates everything? That would be ideal.

Reply:
Looks to me like: ATMAERO was added only.  Seems like a good APP/feature to maybe request for the future?